bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/lgapi/LGThinqApiClientService.java

```diff
@@ -25,6 +21,10 @@
 import org.openhab.binding.lgthinq.internal.errors.LGThinqException;
 import org.openhab.binding.lgthinq.lgapi.model.*;
 
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
```

Comment:
Order imports by name

Reply:
A doubt here, Hans. What is the general correct import layout OpenHab follows ? It's just to reduce come and back works, if you can detail a little bit more here, or send-me a guide to explain better, I would appreciate it! By the way... right know, I'm following this layout:

<static imports>
--- blank like ---
<other imports> (openhab and external packages like apache, etc.)
-- blank line ---
<internal imports> (i.e. not jdk imports)

And, these sessions have the imports ordered by full qualified name (not only the class name). Is that right ?

Reply:
imports are normally organized by IDE (eclipse) automatically or by applying mvn spotless:apply

Reply:
Yeap. This last code was totally formatted the code by spotless:apply. I disabled my IDE's import optimization because it was screwing up the spotless work and the CI/Build was complaining about it. Then I was expecting that import layout was ok, otherwise, I probably have gotten spotless policy check's erros in CI/Build phase, but it passed clean. 
Anyway, I gonna pass spotless:apply again before done the next push. Thanks for support me on weekend.